### PR TITLE
feature/SM-5547: Update Sample Inspection Targets

### PR DESCRIPTION
**Pull request by ccurran01**
## Description

client definition for new update sample inspection endpoint

Street Manager Data: https://github.com/departmentfortransport/street-manager-data/pull/83

## Coding Standards

Style guide: https://github.com/departmentfortransport/street-manager/wiki/Street-Manager-Style-Guide
Coding best practices: https://github.com/departmentfortransport/street-manager/wiki/Street-Manager-Coding-Best-Practices

## Checklist

- [ ] Branch named {feature|hotfix|task}/{SM-.*}
- [ ] If your pull request depends on any other, please link them
- [ ] Changes approved by your team
- [ ] Changes approved by another team
- [ ] API definitions updated
- [ ] Commit messages are meaningful
- [ ] Add `DO NOT MERGE` if you want to postpone merge

### CODE REVIEW

File: src/app/client.ts [hunk 1]
```diff
@@ -349,6 +350,10 @@ export class StreetManagerApiClient {
     return this.httpHandler<SampleInspectionTargetCreateResponse>(() => this.axios.post('/sample-inspection-targets', createSampleInspectionTargetRequest, this.generateRequestConfig(requestConfig)))
   }
 
+  public async updateSampleInspectionTarget(requestConfig: RequestConfig, sampleInspectionTargetReferenceNumber: string, updateSampleInspectionTargetRequest: SampleInspectionTargetUpdateRequest): Promise<void> {
```
Comment (caoimheb) [inline src/app/client.ts:353]: Where do endpoints that exist on latest only get added to the API docs?

File: src/interfaces/sampleInspectionTargetUpdateRequest.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  /** Is whole number between 0 and 100
+   * If not provided, the value will be 0
+   */
+  cap_category_a: number
+  /** Is whole number between 0 and 100
+   * If not provided, the value will be 0
+   */
+  cap_category_b: number
+  /** Is whole number between 0 and 100
+   * If not provided, the value will be 0
```
Comment (caoimheb) [inline src/interfaces/sampleInspectionTargetUpdateRequest.ts:19]: Field is requred, so don't think we need the `If not provided, the value will be 0`?

Comment (ccurran01) [reply src/interfaces/sampleInspectionTargetUpdateRequest.ts:19]: No, we don't. Will remove

File: src/app/client.ts (hunk 1 above)
Comment (ccurran01) [reply src/app/client.ts:353]: I'll check and follow up with a docs PR

**Pull request merged**